src/app/pages/splash/containers/splash-page.component.ts

```diff
@@ -39,6 +39,9 @@ export class SplashPageComponent {
     this.status = this.localization.translateKey(
       LocKeys.SPLASH_STATUS_UPDATING_CONFIG
     )
+    this.splash
+      .isAppUpdateAvailable()
+      .then(res => (res ? this.showAppUpdateAvailable() : []))
```

Comment:
Why use `then` ? As it is returning a `boolean`, you can directly use- 
```ts
if (this.splash.isAppUpdateAvailable()) {
 this.showAppUpdateAvailable()
}
```
or you can combine the above two functions into a single one and just call that. eg - `this.splash.updateAppIfRequired()`

Also please rename `splash` to `splashService` for brevity.

Reply:
Yes, pulling from remote config returns a promise.

Reply:
Ok thanks